src/imguiclient/imguiapp.h

```diff
@@ -13,11 +13,13 @@
 #pragma once
 
 // clang-format off
-#include <imgui_internal.h>
 #include <imgui.h>
+#ifdef _WIN32
+#include <imgui_internal.h>
```

Comment:
Looks like we do not need this anymore. I tested on Windows.

Reply:
Ok I've removed it.